Alarmi/Alarmi/Sources/ViewControllers/SettingAlarm/Supporting Views/AlarmAgainSetView.swift

```diff
@@ -0,0 +1,191 @@
+        $0.axis = .horizontal
+        $0.spacing = 4
+        $0.distribution = .fill
+        $0.alignment = .center
+        return $0
+    }(UIStackView())
+
+    private let titleLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.setDynamicFont(.title2)
+        $0.text = "다시 알림"
+        return $0
+    }(UILabel())
+
+    private let descriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.textColor = .secondaryLabel
+        $0.setDynamicFont(.footnote)
+        $0.text = "알림이 온 이후 전화를 기록하지 않으면,\n알림을 다시 보냅니다."
+        return $0
+    }(UILabel())
+
+    private lazy var alarmSwitch: UISwitch = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.isOn = true
+        $0.addTarget(self, action: #selector(alarmSwitchValueChanged), for: .valueChanged)
+        return $0
+    }(UISwitch())
+
+    private let subTitleLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "횟수"
+        $0.setDynamicFont(.headline)
+        return $0
+    }(UILabel())
+
+    private let hStackView2: UIStackView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.axis = .horizontal
+        $0.spacing = 6
+        $0.distribution = .fill
+        $0.alignment = .center
+        return $0
+    }(UIStackView())
+
+    private let minLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "2"
+        $0.setDynamicFont(.callout)
+        $0.textColor = .secondaryLabel
+        return $0
+    }(UILabel())
+
+    private let maxLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "10"
+        $0.setDynamicFont(.callout)
+        $0.textColor = .secondaryLabel
+        return $0
+    }(UILabel())
+
+    private lazy var slider: UISlider = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.minimumValue = 2
+        $0.maximumValue = 10
+        $0.value = 6
+        $0.addTarget(self, action: #selector(sliderValueChanged), for: .valueChanged)
+        return $0
+    }(UISlider())
+
+    private let sliderLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "최대 6번 알림을 다시 받을게요."
+        $0.textAlignment = .center
+        return $0
+    }(UILabel())
+
+    // MARK: Parameters
+
+    private var alarmAgainCount: Int = 5 {
+        didSet {
+            sliderLabel.text = "최대 \(alarmAgainCount)번 알림을 다시 받을게요."
+        }
+    }
+
+    // MARK: Life Cycle
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        attribute()
+        layout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: Methods
+
+    private func attribute() {
+        backgroundColor = .systemBackground
```

Comment:
p1;

죄송합니다... 반영해주세요🙇‍♂️

```suggestion
        backgroundColor = .secondarySystemGroupedBackground
```